examples/nyc_boros.py

```diff
@@ -1,36 +1,57 @@
 """
-Generate example images for GeoPandas documentation.
+Visualizing NYC Boroughs
+------------------------
 
-TODO: autogenerate these from docs themselves
+Visualize the Boroughs of New York City with Geopandas.
 
-Kelsey Jordahl
-Time-stamp: <Tue May  6 12:17:29 EDT 2014>
+This example generates many images that are used in the documentation. See
+the `Geometric Manipulations <geometric_manipulations>` example for more
+details.
```

Comment:
Maybe, can you move the original one inside the `docs/` directory? 
The we can make this a cleaner one focused as user example